src/Common/src/Interop/Windows/kernel32/Interop.GetComputerName.cs

```diff
@@ -11,6 +12,9 @@ internal partial class Kernel32
         [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, EntryPoint = "GetComputerNameW")]
         private static extern unsafe int GetComputerName(char* lpBuffer, ref uint nSize);
 
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        public static extern bool GetComputerName(StringBuilder lpBuffer, int[] nSize);
```

Comment:
Nit: Instead of adding this new overload that takes a `StringBuilder` for `lpBuffer`, why not use the existing internal helper method below that returns a string (and avoids the `StringBuilder` allocation and marshaling costs)? Then update the calling code in PerformanceCounterLib.cs to something like the following:

```c#
internal static string ComputerName
{
    get
    {
        if (s_computerName == null)
        {
            lock (InternalSyncObject)
            {
                if (s_computerName == null)
                {
                    s_computerName = Interop.Kernel32.GetComputerName() ?? string.Empty;
                }
            }
        }

        return s_computerName;
    }
}
```

Note that the above maintains the previous behavior of setting `s_computerName` to an empty string if the call to `GetComputerName` fails, which could happen previously when `StringBuilder.ToString()` is called on an empty builder.